nemo_curator/scripts/semdedup/clustering.py

```diff
@@ -74,7 +69,7 @@ def main(args):
 
     clustered_embeddings = clustering_model(embedding_dataset)
     clustered_embeddings.df.head(10)
-    dt2 = datetime.now()
+    dt2 = datetime.now()  # noqa: DTZ005
```

Comment:
Instead of `datetime.now()` it could be `time.perf_counter()` like elsewhere in this PR?